packages/yew/src/virtual_dom/listeners.rs

```diff
@@ -66,17 +66,28 @@ macro_rules! gen_listener_kinds {
         /// Supported kinds of DOM event listeners
         // Using instead of strings to optimise registry collection performance by simplifying
         // hashmap hash calculation.
-        #[derive(Clone, Copy, PartialEq, Eq, Hash, Debug)]
+        #[derive(Clone, PartialEq, Eq, Hash, Debug)]
         #[allow(non_camel_case_types)]
         #[allow(missing_docs)]
         pub enum ListenerKind {
             $( $kind, )*
+            other(std::borrow::Cow<'static, str>),
```

Comment:
I believe we should use proper style for this variant and use upper case `Other`. This also conveys that it's different from the other variants.